app/models.py

```diff
@@ -1462,10 +1464,12 @@ def validate(self, enforce_required=True, required_fields=None, max_day_rate=Non
 
     def serialize(self):
         data = self.data.copy()
+        parent_brief = self.brief.serialize()
+        parent_brief_fields = ['id', 'title', 'status', 'applicationsClosedAt']
```

Comment:
Out of interest, why are we adding `lazy= joined`? This eager loading has caused some of our performance issues elsewhere, so unless it's essential it might be better to lazy load if possible.